src/web/tools/GestureHandlerOrchestrator.ts

```diff
@@ -196,12 +218,11 @@ export default class GestureHandlerOrchestrator {
       }
     }
 
-    this.awaitingHandlers.forEach((otherHandler) => {
+    for (const otherHandler of this.awaitingHandlers) {
       if (this.shouldHandlerBeCancelledBy(otherHandler, handler)) {
-        otherHandler?.cancel();
-        otherHandler?.setAwaiting(true);
+        otherHandler.setAwaiting(true);
       }
-    });
+    }
```

Comment:
This loop doesn't make a lot of sense, we go through the awaiting handlers and mark them as awaiting. On Android, we mark them as *not* awaiting and then clean up the awaiting handlers list, which makes more sense since `shouldHandlerBeCancelledBy` is the condition. Because of that, I'm also not sure about removing the call to `.cancel()`, but the logic on web here is not 1:1 to Android.

Reply:
>This loop doesn't make a lot of sense, we go through the awaiting handlers and mark them as awaiting.

I agree. It was there for a very long time, probably we overlooked it during development of new web implementation. I've changed that to `false` and it seems to work fine. 

>Because of that, I'm also not sure about removing the call to .cancel(), but the logic on web here is not 1:1 to Android.

I think that removing `cancel` is fine, since this is the same condition as in the loop above and we agreed, that if handler is in `gestureHandlers` then it has to be in `awaitingHandlers` (that's why we also removed this `cancel` on android).

Done in [cca5d5f](https://github.com/software-mansion/react-native-gesture-handler/pull/2819/commits/cca5d5fa776cccc43e3135fac5e0b05071b81d3c).